data/README.md

```diff
@@ -0,0 +1,9 @@
+# Simorgh Data Samples
+
+These are a set of data samples which represent possbile input to Simorgh from upstream systems (and ultimately from a CMS).  They cover a number of scenarios, most of which are made up with random Lorem Ipsum or real text.
+
+Two of the scenarios, 25 and 27, are based on real articles published on the BBC News website.
+
+Some of the scenarios are intended to represent failures - where we would reject the articles as being malformed.
```

Comment:
Which?

Reply:
It wouldn't be a blind test if I told you that :P